### PR TITLE
XW-1619 | add list component styles

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://github.com/Wikia/design-system-reference-page/pull/64

Duplicate for https://github.com/Wikia/design-system/pull/33, but target branch is `master` here

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: components/_lists.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	margin: 18px;
+	padding: 0;
+}
+
+.wds-list-item {
+	display: flex;
+	flex-direction: column;
+	font-size: $typescale-size-minus-1;
+	justify-content: center;
+	min-height: 36px;
```
Comment (kvas-damian) [inline components/_lists.scss]: Should we use `min-height` or `padding` here? What would happen if there is a multiline item?

**Comment by kvas-damian** [comment]
What about list with separators? Weren't we planning to implement it as part of this ticket?

File: components/_lists.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.wds-list {
+	display: block;
+	list-style-type: none;
+	margin: 18px;
+	padding: 0;
+}
+
+.wds-list-item {
+	display: flex;
```
Comment (Warkot) [inline components/_lists.scss]: Using flex for an item makes all tags in the item to be displayed as a separate flex item. Putting even an inline element such as `<a>` inside the list item will break the line.

File: components/_lists.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+	> li {
+		box-sizing: border-box;
+		font-size: $typescale-size-minus-1;
+		line-height: 1em;
+		padding: 11px 0;
+	}
+
+	&.wds-has-big-items {
+		margin: 2px 18px;
```
Comment (rogatty) [inline components/_lists.scss]: Or:
```
margin: {
  bottom: 2px;
  top: 2px;
}

File: components/_lists.scss [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+		> li {
+			font-size: $typescale-size-base;
+			padding: 16px 0;
```
Comment (rogatty) [inline components/_lists.scss]: Or:
```
padding: {
  bottom: 16px;
  top: 16px;
}
```

**Comment by rogatty** [comment]
Minors but overall LGTM

**Comment by Warkot** [comment]
LGTM

**Pull request merged**